ui/src/components/ripple-out/RippleOutStoryContainer.js

```diff
@@ -33,111 +108,62 @@ const RippleOutStoryContainer = () => {
           h="100%"
           textStyle="darker_grotesque"
         >
-          <Flex flexDir="column" gap={9}></Flex>
-          <VStack gap="4em" textAlign="center">
+          <Text fontSize={headerFontSize} fontWeight={800}>
+            THE STORY
+          </Text>
+          <VStack mt={['2.5em', '5em']} gap="4em" textAlign="center">
             <Text
-              fontSize={['3em', '4.75em']}
-              lineHeight="1em"
+              fontSize={['2.25em', '2.75em', '3.25em', '3.75em', '4.25em']}
+              lineHeight={['1.5em', '1.5em']}
               textStyle="darker_grotesque_medium"
             >
-              We are called to <u>RIPPLE OUT</u> as we live out our{' '}
-              <u>SATURATE VISION</u> to saturate Hong Kong with the knowledge of
-              God’s glory
-            </Text>
-            <VStack lineHeight="2em">
-              <Text fontSize={['3xl', '4xl']}>
-                TRANSFORMING LIVES TRANSFORMING THE WORLD
-              </Text>
-              <Text
-                fontSize={['3xl', '4xl']}
-                textStyle="darker_grotesque_medium"
+              <Highlight
+                query={['ripple out', 'saturate vision']}
+                styles={{
+                  px: '1',
+                  bg: '#D7BB84',
+                  fontWeight: '800',
+                  color: '#182E57',
+                }}
               >
-                Transformation Center {'>>'} Kwun Tong {'>>'} Hong Kong {'>>'}{' '}
-                Ends of the earth
-              </Text>
-            </VStack>
-
-            <Button
-              variant="outline"
-              onClick={onOpen}
-              fontSize="lg"
-              bgColor="#ffffff"
-              fontWeight="bold"
-              borderColor="#182E57"
-              color="#182E57"
+                We are called to ripple out as we live out our Saturate vision
+                to saturate Hong Kong with the knowledge of God’s glory
+              </Highlight>
+            </Text>
+            <Image
```

Comment:
<img width="385" alt="image" src="https://github.com/hmcc-global/hmcchk-web/assets/16095743/19f5abd5-2044-405c-b790-9985b9e0a9a1">

so the sides of the image gets affected by the padding from the container. i think u might need to take the image out of the container so that it can break the margin rules of the container cos according to the design this is edge to edge on the screen

Reply:
also can u confirm with CXO if they want this to be a gif, saw that on figma but not sure

Reply:
no gif!